redaxo/src/core/lib/packages/manager.php

```diff
@@ -136,14 +136,15 @@ public function install($installDump = true)
             rex_autoload::addDirectory($this->package->getPath('lib'));
             rex_autoload::addDirectory($this->package->getPath('vendor'));
             rex_i18n::addDirectory($this->package->getPath('lang'));
-
+            $instmsg = '';
             // include install.php
             if (is_readable($this->package->getPath(rex_package::FILE_INSTALL))) {
                 $this->package->includeFile(rex_package::FILE_INSTALL);
-
-                if ('' != ($instmsg = $this->package->getProperty('installmsg', ''))) {
-                    throw new rex_functional_exception($instmsg);
+                $instmsg = $this->package->getProperty('installmsg', '');
+                if ('' !== $instmsg) {
+                    $instmsg = '<p>'.$instmsg.'</p>';
                 }
+
```

Comment:
auf diese art und weise können wir das schon vor diesem PR existierende property nicht neu definieren.

bereits existierende Addons könnten bereits darauf vertrauen dass das setzen einer `installmsg` dazu führt dass die installation abgebrochen wird.

ich denke wir müssen ein neues property einführen

Reply:
ich denke da z.B. an `statusmsg` statt `installmsg`

Reply:
ja mich hat generell gewundert wieso da eine exception kommt wenn eine install message definiert ist, ist das vielleicht sogar dafür gedacht gewesen dass man bei der Installation eine eigene error Meldung einbauen konnte?

Wie soll die neue Property heißen?

Reply:
> ja mich hat generell gewundert wieso da eine exception kommt wenn eine install message definiert ist, ist das vielleicht sogar dafür gedacht gewesen dass man bei der Installation eine eigene error Meldung einbauen konnte?

ja dafür ist es urspr. eingebaut worden. es existiert seit vielen jahren in der form. heute würden wir es vermutlich nicht mehr so bauen ;-)

> Wie soll die neue Property heißen?

`statusmsg`

Reply:
alles klar ich fixe das mal schnell